.github/workflows/sync-ruby.yml

```diff
@@ -0,0 +1,33 @@
+name: Sync ruby
+on:
+  push:
+    branches: [master]
```

Comment:
On mobile so I can't fix this now, but I'm pretty sure this should be [main]

Reply:
yeah sorry, I've fixed that one https://github.com/ruby/prism/commit/c89ca2af12ba20b4fd2c5ff43ebe25da1d81d8db